ihp-sg13g2/libs.ref/sg13g2_io/liberty/sg13g2_io_dummy.lib

```diff
@@ -86,4 +86,2956 @@ library (sg13g2_io_dummy) {
+                    values ( \
+                        "200, 200", \
+                        "200, 200" \
+                    );
+                }
+            }
+                }    }
+
+    cell (IOPadTriOut4mA) {
+        area: 14400;
+        dont_touch : true;
+        dont_use : true;
+        timing_model_type : abstracted;
+        pad_cell : true;
+        pg_pin (vss) {
+            pg_type : primary_ground;
+            voltage_name : "vss";
+        }
+        pg_pin (vdd) {
+            pg_type : primary_power;
+            voltage_name : "vdd";
+        }
+        pg_pin (iovss) {
+            pg_type : primary_ground;
+            voltage_name : "iovss";
+        }
+        pg_pin (iovdd) {
+            pg_type : primary_power;
+            voltage_name : "iovdd";
+        }
+        pin (c2p) {
+            direction : input;
+            input_voltage : core_input;
+            related_ground_pin : vss;
+            related_power_pin : vdd;
+            max_transition : 200;
+            capacitance : 15.0;
+            rise_capacitance : 15.0;
+            rise_capacitance_range (12.0, 15.0);
+            fall_capacitance : 15.0;
+            fall_capacitance_range (12.0, 15.0);
+        }
+        pin (c2p_en) {
+            direction : input;
+            input_voltage : core_input;
+            related_ground_pin : vss;
+            related_power_pin : vdd;
+            max_transition : 200;
+            capacitance : 15.0;
+            rise_capacitance : 15.0;
+            rise_capacitance_range (12.0, 15.0);
+            fall_capacitance : 15.0;
+            fall_capacitance_range (12.0, 15.0);
+        }
+        pin (pad) {
+            direction : output;
+            function : "(c2p)";
+            three_state: "(c2p_en)";
+            is_pad : true;
+            output_voltage : io_output;
+            related_ground_pin : iovss;
+            related_power_pin : iovdd;
+            max_capacitance : 500;
+            max_transition : 200;
+            timing () {
+                related_pin : "c2p";
+                timing_type : combinational;
+                cell_rise (delay_template_2x2) {
+                    values ( \
+                        "1000, 1000", \
+                        "1000, 1000" \
+                    );
+                }
+                rise_transition (delay_template_2x2) {
+                    values ( \
+                        "200, 200", \
+                        "200, 200" \
+                    );
+                }
+                cell_fall (delay_template_2x2) {
+                    values ( \
+                        "1000, 1000", \
+                        "1000, 1000" \
+                    );
+                }
+                fall_transition (delay_template_2x2) {
+                    values ( \
+                        "200, 200", \
+                        "200, 200" \
+                    );
+                }
+            }
+            timing () {
```

Comment:
I don't see any obvious issue here with these two timings. I wonder if timings are missing in the pin c2p and pin c2p_en definitions...